content/docs/contributing/to-docs/get-started.md

```diff
@@ -1,108 +1,252 @@
+{{< /important >}}
+
+### Preview your docs
+
+As you write, you should preview of your docs to ensure it's in the correct format. Because the o3de.org website is built on Hugo, the best way to preview your docs is by running Hugo on your local machine. For information on how to set up and use Hugo, refer to the o3de.org [`README.md`](https://github.com/o3de/o3de.org) file. 
+
+Additionally, a quick way to preview your docs is by using VS Code's [Editor and preview synchronization](https://code.visualstudio.com/docs/languages/markdown#_editor-and-preview-synchronization) feature for Markdown. This feature is limited to VS Code's Markdown support, so it won't render any features that are unique to Hugo, such as shortcodes. 
+
+Finally, you can preview and share a preview of your docs when they are in a PR. Netlify deploys a preview for each PR. This takes a few minutes to deploy after creating the PR, and refreshes everytime you push a commit. You can access the deployed preview at the bottom of the PR webpage: Find **netlify/o3deorg/deploy-preview — Deploy Preview ready!** and click **Details**. 
+
+
+## Submitting docs
+
+When you make changes or create new docs, you must create a pull request (PR) for review before the changes can be merged into `o3de.org:main`. PRs allow peer contributors to review contributions for several potential issues, including technical accuracy, spelling, grammar, clarity, and style. 
+
+The current list of active PRs is here [O3DE repository pull requests (PRs)](https://github.com/o3de/o3de.org/pulls).
+
+
+### Create a PR
+
+PRs are created in the GitHub web interface from your branch. Go to your fork on GitHub and perform the steps below.
+
+1. In the **Pull Request** tab, select the green **New pull request** button to create a new PR.
+
+1. In the **Comparing changes** page, you will compare your fork's branch against the remote source's branch. Ensure that the **base**, which you're comparing against, points to `o3de/o3de.org:main` or `o3de/o3de.org:development`. The **head repository** should be your fork. Then, in the **compare** dropdown, choose the branch that you want to submit.
+
+1. Verify that the changed files contains only the changes you want to submit. Only the commits you made should be listed. If there are more commits than you expected, stop here and make sure to properly sync your branch.
```

Comment:
There are several sections describing how to sync in this topic. A link to these specific iteration steps that you introduced earlier would help clarify here:
```suggestion
1. Verify that the changed files contains only the changes you want to submit. Only the commits you made should be listed. If there are more commits than you expected, stop here and make sure to properly sync your branch using the fetch, merge, and push commands from the [iteration steps](#sync-write-add-commit-and-push) explained earlier.
```